### ISSUE TITLE
[BUG] Hover states in UI are likely not touch device or screen-reader friendly

### ISSUE THREAD

**Issue by benjaminfh** [benjaminfh/researchtree#456]
We use hover states in various parts of the UI. Touch devices won't behave intuitively in all cases. We may also be lacking screen reader support.

Acceptance criteria:
- Where possible, we have graceful handling of hover states on touch devices (e.g. considering https://medium.com/@afron.orana/handling-hover-state-on-touch-devices-58d4851f2e21)
- Where not possible, we have thoughtful fallback UI behaviour
- Buttons have labels that are valid for screen readers
